apps/svelte.dev/content/docs/kit/30-advanced/40-service-workers.md

```diff
@@ -64,7 +64,7 @@ self.addEventListener('fetch', (event) => {
 		const url = new URL(event.request.url);
 		const cache = await caches.open(CACHE);
 
-		// `build`/`files` can always be served from the cache
+		// `build` / `files` can always be served from the cache
```

Comment:
Presumably not strictly necessary here. I don't know whether this still makes sense to do anyway.

Reply:
I think we may as well be consistent